README.md

```diff
@@ -66,6 +66,52 @@ scripts/test
 scripts/test -w
 ```
 
+## Funding codes
+
+In order to persist payment request data and construct a statement dataset, the service must be populated with a mapping of funding option codes to a name.
+These values are stored in the `fundingOptions` PostgreSQL database table.
+
+### SFI Pilot
+
+| Name | Code |
```

Comment:
would make the names case identical to the records in the db

Reply:
Serves me right for copy and pasting the Pilot ones...

Fixed now.